documentation/self-host/community-edition/install-and-build.mdx

```diff
@@ -21,10 +21,14 @@ DATABASE_URL=postgresql://username:password@url:5432/dbname # or replace with yo
 JWT_SECRET=secretcode123
 TOKEN_SALT_COMPLEXITY=10
 MAGIC_LINK_TOKEN_VALIDITY=3
-REFRESH_TOKEN_VALIDITY=604800000
-ACCESS_TOKEN_VALIDITY=86400000
+REFRESH_TOKEN_VALIDITY=604800000 # Default validity is 7 days (604800000 ms) in ms
+ACCESS_TOKEN_VALIDITY=86400000 # Default validity is 1 day (86400000 ms) in ms
 SESSION_SECRET=anothersecretcode123
 
+# Reccomended to be true. Set to false if you are using http
+# Note: Some auth providers may not support http requests and may stop working when set to false
+ALLOW_SECURE_COOKIES=true
+
```

Comment:
Lets not merge this part just yet, the PR is still open and its not merged.

Reply:
Cool